growthbook/growthbook
# Ji/create clickhouse datasource

### Features and Changes

Creates an inbuilt Growthbook Clickhouse option

### Testing

- Create new datasource
- Click on the create in built datasource button
- Click Finish
- Edit the anonymous_id assignment query on the datasource page 
- Click Test Query
- See 4 results (`test_enriched_events` needs to have events that match the org in your database)


### Screenshots

![Screenshot 2024-10-17 at 6 50 22 PM](https://github.com/user-attachments/assets/1dca26ed-8971-4d63-a905-49fd3fce3ea1)
when the sample database is also showing:
![Screenshot 2024-10-17 at 6 50 01 PM](https://github.com/user-attachments/assets/89ce16b7-8310-445b-bc42-d5618f7b3570)
